lib/chef_zero/server.rb

```diff
@@ -111,7 +111,7 @@ module ChefZero
   class Server
 
     DEFAULT_OPTIONS = {
-      :host => '127.0.0.1',
+      :host => ['127.0.0.1'],
```

Comment:
Travis just dropped IPv6 local host because GCE doesn't support it. Insert your favorite face palm reaction pic. 

> On Mar 22, 2016, at 9:25 PM, Lamont Granquist <notifications@github.com> wrote:
> 
> In lib/chef_zero/server.rb:
> 
> > @@ -111,7 +111,7 @@ module ChefZero
> >    class Server
> >  
> >      DEFAULT_OPTIONS = {
> > -      :host => '127.0.0.1',
> > +      :host => ['127.0.0.1'],
> in this day and age should ::1 get dropped in here by default or are we not there yet?
> 
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly or view it on GitHub
>